StartTest(t => {

    t.it("Sanity", t => {
        throw "Going to fire the 'testfailedwithexception' event"
    })
})
